tests/boot/boot_from_pxe.pm

```diff
@@ -118,7 +121,8 @@ sub run {
     send_key 'ret';
     save_screenshot;
 
-    if (get_var('ARCH') eq 'aarch64') {
+    check_screen 'orthos-grub-boot-continue', timeout => 60;
```

Comment:
Please use a multi-tag assert_screen with match_has_tag to not waste time

Reply:
> Please use a multi-tag assert_screen with match_has_tag to not waste time

This an optional match, no exception will be thrown out if failed to match. So I have to use check_screen. If so, do you mean if (check_screen 'orthos-grub-boot-continue') ? Instead of check_screen firstly, then if (match_has_tag) again ?

Reply:
No. he means to assert on 2 screens - either with or without orthos. This way there is a defined end to the call - while check_screen sleeps in doubt for 60 seconds, wasting time.

Reply:
> No. he means to assert on 2 screens - either with or without orthos. This way there is a defined end to the call - while check_screen sleeps in doubt for 60 seconds, wasting time.

I will think around.